plugins/destination/postgresql/client/write.go

```diff
@@ -20,6 +21,14 @@ var cqStatusToPgStatus = map[schema.Status]pgtype.Status{
 	schema.Present:   pgtype.Present,
 }
 
+type pgErrorWrap struct {
+	*pgconn.PgError
+}
+
+func (p *pgErrorWrap) Error() string {
+	return p.PgError.Error() + " (Table: " + p.PgError.TableName + ")"
```

Comment:
We can also reimplement instead of calling the base `Error()`. Reimplementing might be more robust